### DISCUSSION TITLE
Why do PR's get marked with Needs: Attention that have passed azure pipelines for no apparent reason

### DISCUSSION THREAD

**Discussion by AwesomeKalin** [microsoft/winget-pkgs#9974] [Q&A] [answered]
Title

**Comment by denelon** [comment]
@AwesomeKalin can you give a link to a PR. We might have a bug in our code, or an upstream dependency we need to be more resilient about.

**Reply by AwesomeKalin** [reply]
It's not a bot. It's a real person:
https://github.com/microsoft/winget-pkgs/pull/9780
https://github.com/microsoft/winget-pkgs/pull/9754
https://github.com/microsoft/winget-pkgs/pull/9753
https://github.com/microsoft/winget-pkgs/pull/9752

Here are the links

**Comment by KevinLaMS** [comment, marked as answer]
Sorry about that.  There is an issue with the validation pipeline.  The issue is a 'false failure'.  So I flag the PR as needs: attention so that I can verify and approve the PR.
